File: Week 1/lecture/calculator.c
# include "..\cs50.h"
# include <stdio.h>

int main(void){
    // This is a comment. adds values from user
    int first_number = get_int("x: ");
    int second_number = get_int("y: ");
    int z = first_number + second_number;
    printf("%i\n", z);
}

File: Week 6/lecture/calculator.py
from cs50 import get_int

x = get_int("x: ")
y = get_int("y: ")

z = x/y
z2 = x//y
print(z)
print(z2)
print(f"{z}")
print(f"{z:.50f}") # 50 digits after dp, this is a comment